core/templates/components/state-editor/state-editor-properties-services/state-hints.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for the State Hints service.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { UtilsService } from 'services/utils.service';
+import { AlertsService } from 'services/alerts.service';
+/* eslint-disable max-len */
+import { StateHintsService } from 'components/state-editor/state-editor-properties-services/state-hints.service';
```

Comment:
```suggestion
import { StateHintsService } from
  'components/state-editor/state-editor-properties-services/state-hints.service';
```

Reply:
And remove `eslint-disable max-len` and `eslint-enable max-len`

Reply:
still showing an error.

Reply:
Hi @safwan20, you can break the file path using "+". (see other files).

Reply:
@kevintab95 I tried this. even I looked up other files they too have used
 /* eslint-disable max-len */

Reply:
That's weird, it should not error if you break the filepath such that it does not cross the 80 char limit. Do you mean it failed to compile or errored?

Also, I think we should get rid of the `/* eslint-disable max-len */` if it is being used in other places for import statements.

Reply:
I think it's not possible to use `+` when using `import`, just `require`, I've faced something similar before. I think it's ok to use `/* eslint-disable max-len */` in this case

Reply:
Oh okay, makes sense to use the pragma in that case. Thanks for clarifying! :)

Reply:
Don't mention it!